From 745034c6acc386fe9e7d3d4b0497a945c87f4d2a Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Thu, 2 Oct 2025 20:34:28 +0200
Subject: [PATCH] Depend on zlib on Windows

---
 mariadb_windows.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mariadb_windows.py b/mariadb_windows.py
index 034df10..82ad926 100755
--- a/mariadb_windows.py
+++ b/mariadb_windows.py
@@ -69,7 +69,7 @@ def get_config(options):
                     mariadb_dir + "\\include\\mariadb\\mysql"]
     cfg.lib_dirs = [mariadb_dir + "\\lib", mariadb_dir + "\\lib\\mariadb"]
     cfg.libs = ["ws2_32", "advapi32", "kernel32", "shlwapi", "crypt32",
-                "secur32", "bcrypt"]
+                "secur32", "bcrypt", "zlib"]
     if static.lower() == "on" or static.lower() == "default":
         cfg.libs.append("mariadbclient")
     else:
